core/lib/src/response/responder.rs

```diff
@@ -217,34 +219,142 @@ impl<'r> Responder<'r> for String {
     }
 }
 
+fn try_range_response<'r, B: io::Read + io::Seek + 'r>(req: Request<'r>, mut body: B) -> Result<response::Response<'r>, B> {
+    use std::cmp;
+    use http::hyper::header::{ContentRange, ByteRangeSpec, ContentRangeSpec};
```

Comment:
Is `typed-headers` an option for Rocket / are you planning to use this crate? In that case I might try to contribute an implementation for these headers in `typed-headers` instead of adding this directly to Rocket.

Reply:
It looks like `RangeUnit` and `AcceptRanges` were previously in `typed-headers` and removed, but `Range` was never present. I think I'm okay with using hyper 0.10's typed headers for now, and crossing that bridge when we come to it.